src/lib/components/WellCompletions/components/Settings/HideZeroCompletionsSwitch.tsx

```diff
@@ -19,7 +19,6 @@ const HideZeroCompletionsSwitch: React.FC = React.memo(() => {
         <Switch
             label="Filter by completions"
             size="small"
-            enterKeyHint="Only show wells completed in at least one of the selected layers"
```

Comment:
This is the text that would appear instead of the `enter` key on a virtual keyboard, if the component was a text field, and it has to be a value from a specific set. See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/enterkeyhint

Reply:
Thanks! I might have mistaken it with a tooltip hint. LGTM now.